### PR TITLE
feat: moves "Using password from..." to stderr

### PR DESCRIPTION
**Pull request by kobigurk**
This is a small change to move the "Using password from..." message from stdout to stderr. This message complicates automation since it has to filter for it, e.g. automatic restarts, vrf commands, metrics.

### CODE REVIEW

File: automation/services/watchdog/metrics.py
Comment (mrmr1993) [inline automation/services/watchdog/metrics.py:120]: This change seems unrelated. Can you remove it?

**Review by mrmr1993 — CHANGES_REQUESTED** [review]

File: automation/services/watchdog/metrics.py
Comment (kobigurk) [reply automation/services/watchdog/metrics.py:120]: It's related, since it was a workaround due to this :)

Comment (kobigurk) [reply automation/services/watchdog/metrics.py:120]: But happy to remove.

Comment (mrmr1993) [reply automation/services/watchdog/metrics.py:120]: It still seems unrelated, but please feel free to explain further.

Comment (kobigurk) [reply automation/services/watchdog/metrics.py:120]: Line 121 says it: `#first line could be 'Using password from environment variable MINA_PRIVKEY_PASS'`

Comment (kobigurk) [reply automation/services/watchdog/metrics.py:120]: But you're right, maybe best to not introduce this change for now.

Comment (kobigurk) [reply automation/services/watchdog/metrics.py]: Alright, removed it!

**Review by mrmr1993 — APPROVED** [review]

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by kobigurk** [comment]
@mrmr1993 can we run it again? seems like a possible intermittent error

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
> @mrmr1993 can we run it again? seems like a possible intermittent error

@kobigurk I think that PR #9383 is supposed to fix this error; once that's merged you should be able to merge `compatible` to this branch and it'll work again.

**Comment by mrmr1993** [comment]
> @kobigurk I think that PR #9383 is supposed to fix this error

It's landed now, you should be good to go if you merge `compatible`

**Comment by kobigurk** [comment]
> > @kobigurk I think that PR #9383 is supposed to fix this error
> 
> It's landed now, you should be good to go if you merge `compatible`

Rebased!

**Review by lk86 — APPROVED** [review]

**Pull request merged**
